source/deployment/bulk-loading-data-format.rst

```diff
@@ -79,8 +79,8 @@ For clarity, the object is shown using regular JSON formatting, but in the data
 .. code-block:: javascript
 
   {
-    type: "version",
-    version: 1
+    "type": "version",
+    "version": "1"
```

Comment:
Actually numbers don't have to be in quotes.

Reply:
Thanks, have fixed :)